tests/test_telcameras_v2_ingress.py

```diff
@@ -24,6 +27,61 @@ def setUp(self):
         # Create an endpoint
         self.endpoint_obj = Endpoint.objects.create(url_key=self.endpoint_url_key)
 
+        # Create the sensor in the database
+        self.sensor = Sensors.objects.create(objectnummer='GAVM-01-Vondelstraat')
+
+    def test_data_for_non_existing_sensor_is_not_added_to_the_db(self):
+        # First add a couple ingress records with a non existing sensor code
+        IngressQueue.objects.all().delete()
+        post_data = json.loads(TEST_POST)
+        post_data['data'][0]['sensor'] = 'does not exist'
+        for i in range(3):
+            self.client.post(self.URL, json.dumps(post_data), **AUTHORIZATION_HEADER, content_type='application/json')
+        self.assertEqual(IngressQueue.objects.count(), 3)
+
+        # Then run the parse_ingress script
+        out = call_man_command('parse_ingress')
+        self.assertEqual(out.strip(), "Parsed: 3 Success: 3")
+
+        # Test whether the records in the ingress queue are correctly set to parsed
+        self.assertEqual(IngressQueue.objects.all().count(), 3)
+        for ingress in IngressQueue.objects.all():
+            self.assertIsNotNone(ingress.parse_started)
+            self.assertIsNotNone(ingress.parse_succeeded)
+            self.assertIsNone(ingress.parse_failed)
+
+        # Test whether the records were indeed not added to the database
+        self.assertEqual(Observation.objects.all().count(), 0)
```

Comment:
Dit voelt alsof het niet klopt; we parsen data, zeggen dat alles success is, maar slaan niks op.